packages/components/package.json

```diff
@@ -90,7 +90,8 @@
   "scripts": {
     "start": "pnpm dev",
     "dev": "storybook dev -p 6006 --no-version-updates",
-    "build:ts": "rm -rf dist/src dist/tsconfig.build.tsbuildinfo && tsc -b",
+    "clean": "rm -rf dist",
+    "build:ts": "tsc -b",
```

Comment:
Now we don't clean up `dist` dir on builds; in CI it doesn't matter because it always starts from scratch, but in local dev you might have to manually run `pnpm clean` if you move `src/Foo.tsx` to `src/Foo/index.tsx`.